concourse/pipelines/certification_pipeline.yml

```
@@ -357,10 +422,31 @@ jobs:
       - Certify GPDB-6 with PXF-GP6 on Ubuntu 18.04
     - get: ccp-7-image
   - task: Print Report for GPDB-6 with PXF-GP6 Artifacts
-    <<: *slack_alert
     file: pxf_src/concourse/tasks/certification_list.yml
     image: ccp-7-image
     params:
       GOOGLE_CREDENTIALS: ((ud/pxf/secrets/pxf-storage-service-account-key))
       GP_VER: 6
       PXF_CERTIFICATION_FOLDER: data-gpdb-ud-pxf-build/prod/certifications
+
+- name: Reporting Gate for PXF-GP7
+  plan:
+  - in_parallel:
+    - get: pxf_src
+    # gpdb release candidate tarballs and PXF RPMs used in testing jobs
+    - get: gpdb7_tarball_rhel8
+      passed:
+      - Certify GPDB-7 with PXF-GP7 on RHEL8
+      trigger: true
+    - get: pxf_gp7_rpm_rhel8
+      passed:
+      - Certify GPDB-7 with PXF-GP7 on RHEL8
+      trigger: true
+    - get: ccp-7-image
+    - task: Print Report for GPDB-7 with PXF-GP7 Artifacts
+      file: pxf_src/concourse/tasks/certification_list.yml
+      image: ccp-7-image
+      params:
+        GOOGLE_CREDENTIALS: ((ud/pxf/secrets/pxf-storage-service-account-key))
+        GP_VER: 6
```

```suggestion
        GP_VER: 7
```
if only for consistency